appcelerator/titanium_mobile
# chore: update APSAnalytics

- Upodate both Android and iOS APSAnalytics
  - Improve event caching behaviour

##### TEST CASE
- Go offline (Turn wifi and mobile data off)
- Press _SEND_
- Close app
- Go online
- Open app
- See less than 1500 events logged
```JS
const win = Ti.UI.createWindow({ backgroundColor: 'white' });
const btn = Ti.UI.createButton({ title: 'SEND' });

btn.addEventListener('click', _ => {
    for (let i = 0; i < 1500; i++) {
        Ti.Analytics.featureEvent(`event_${i}`);
    }
});

win.add(btn);
win.open();
```

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-28403)
